api/framer-motion.api.md

```diff
@@ -843,7 +843,7 @@ export function useAnimation(): AnimationControls;
 export function useCycle<T>(...items: T[]): CycleState<T>;
 
 // @public
-export function useDomEvent(ref: RefObject<Element>, eventName: string, handler?: EventListener | undefined, options?: AddEventListenerOptions): void;
+export function useDomEvent(ref: RefObject<EventTarget>, eventName: string, handler?: EventListener | undefined, options?: AddEventListenerOptions): void;

```

Comment:
Why did this change? Also the optional argument and undefined seem a bit double to me.

Reply:
@niels I think this is a rebuilt API from another external PR that changed the types to more accurately reflect valid arguments (like window and document)

Reply:
Yep that's it! Threw me for a minute too!